src/Sql/dbo/Stored Procedures/SsoConfig_ReadByIdentifier.sql

```diff
@@ -0,0 +1,13 @@
+﻿CREATE PROCEDURE [dbo].[SsoConfig_ReadByIdentifier]
+@Identifier NVARCHAR(50)
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
```

Comment:
Add `TOP 1` here to short-circuit select (since there's no unique constraint index on `[Identifier]`